docs/installation.md

```diff
@@ -8,19 +8,18 @@ The `wego` command-line interface is currently supported on Mac (x86 and Arm), a
 To install the `wego` CLI, please follow the following steps:
 
 ```console
-curl -L https://github.com/weaveworks/weave-gitops/releases/download/v0.2.1/wego-$(uname)-$(uname -m) -o wego
+curl -L "https://github.com/weaveworks/weave-gitops/releases/download/v0.2.1/wego-$(uname)-$(uname -m)" -o wego
```

Comment:
should we not update to 0.2.2 here and in the suggested output?

Reply:
It looks like the site already takes care of that:

![Screenshot from 2021-07-29 06-46-53](https://user-images.githubusercontent.com/2802257/127503413-7bdeb73c-ab62-4253-83dc-17bf836f4bcd.png)

Reply:
magic

Reply:
folks could still come directly to the readme tho, assume that would still reference 0.2.1? Or is there more wizardry i'm not seeing...

Reply:
I think I needed to change it there as well. Updated.